test/test_color.py

```diff
@@ -38,22 +39,26 @@ def test_epsg_fail():
 pixel_per_meter=0.1
 
 
+@pytest.mark.geoportail
 def test_download_image_ok():
     color.download_image_from_geoportail(epsg, layer, minx, miny, maxx, maxy, pixel_per_meter, OUTPUT_FILE, 15)
 
 
+@pytest.mark.geoportail
 def test_download_image_raise1():
     retry_download = color.retry(2, 5)(color.download_image_from_geoportail)
     with pytest.raises(requests.exceptions.HTTPError):
         retry_download(epsg, "MAUVAISE_COUCHE", minx, miny, maxx, maxy, pixel_per_meter, OUTPUT_FILE, 15)
 
 
+@pytest.mark.geoportail
 def test_download_image_raise2():
     retry_download = color.retry(2, 5)(color.download_image_from_geoportail)
     with pytest.raises(requests.exceptions.HTTPError):
         retry_download("9001", layer, minx, miny, maxx, maxy, pixel_per_meter, OUTPUT_FILE, 15)
 
 
+@pytest.mark.geoportail
```

Comment:
This test mocks connection. So it does not connect to Geoportail

You can remove the marker `@pytest.mark.geoportail`